index.js

```diff
@@ -15,11 +15,23 @@ import {CredentialsContainer} from './CredentialsContainer.js';
 import {PermissionManager} from './PermissionManager.js';
 import {WebCredential} from './WebCredential.js';
 
+const DEFAULT_MEDIATOR = 'https://beta.authn.io/mediator';
+
 let loaded;
 export async function loadOnce(mediatorUrl) {
   if(loaded) {
     return loaded;
   }
+
+  if(!mediatorUrl) {
+    if(typeof window === 'undefined') {
+      throw new Error(
+        '"mediatorUrl" is required; cannot use default URL with "location.origin".');
```

Comment:
Come to think of it, the `load()` logic below requires `document`, `navigator` and `window` globals anyway, so I can remove that check and error.